widgetastic_manageiq/__init__.py

```diff
@@ -5447,6 +5448,17 @@ def get_progress_percent(self, plan_name):
         ]
         return dict(list(zip(desc, val)))
 
+    def is_plan_failed(self, plan_name):
```

Comment:
@Yadnyawalkya why do we need `is_plan_failed` and `is_plan_succeeded`? I would also expect this to always return a boolean (and not error_text). It is consistent with our framework to have this be the case (e.g. `is_displayed`).